rails/app/assets/stylesheets/components/welcome.scss

```diff
@@ -8,6 +8,18 @@
   align-items: center;
   text-align: center;
   
+  .logo {
+  
+		 justify-content: center;
+		align-items: center;
+		text-align: center;
```

Comment:
The above 3 styles are inherited from `.welcome` in the markup; you can rely on those instead.

Reply:
Perfect. Fixed.